### PR TITLE
Add Authorization FAQ

### PR DESCRIPTION
**Pull request by litonico**
Adding this to the docs as Richard and I discussed, but feel free to move it elsewhere.

### CODE REVIEW

File: docs/content/any/getting-started/authorization-faq/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+It's always best to build authorization around your existing infrastructure.
+
+If you already maintain a very performant way of exchanging data, like gRPC, you can use that to share authorization data between services.
+
+If not, the simplest approach is to put your authorization data (like roles) in a JSON Web Token.
+This approach works well, but tokens have size limits.
+Your app may outgrow web tokens quickly.
+
+The second-simplest method—and the most robust—is to use an authorization service like Oso Cloud.
+We go over each of your options in detail in our article on [Why Authorizaion is Hard](https://www.osohq.com/post/why-authorization-is-hard).
```
Comment (samscott89) [inline docs/content/any/getting-started/authorization-faq/index.md:76]:
```suggestion
We go over each of your options in detail in our article on [Why Authorization is Hard](https://www.osohq.com/post/why-authorization-is-hard).
```

(drive-by typo spot as skimming this!)

**Review by kreuter — APPROVED** [review]

**Comment by kreuter** [comment]
We're going to put this content elsewhere.
